openstudiocore/src/utilities/core/Path.hpp

```diff
@@ -22,11 +22,12 @@
 
```

Comment:
All these things are mostly free function helpers that could be added into an OpenStudio Path class or a single header with free helper methods if we don't wrap filesystem